yaptide/admin/db_manage.py

```diff
@@ -172,11 +187,25 @@ def list_tasks():
 
 @run.command
 @click.option('-v', '--verbose', count=True)
-def list_simulations(verbose):
+@click.option('user','--user', default='')
+@click.option('auth_provider', '--auth-provider')
+def list_simulations(verbose, user, auth_provider):
     """List simulations"""
+    if bool(auth_provider) != bool(user):
+        raise click.UsageError("Both --user and --auth-provider must be provided together or neither.")
```

Comment:
I don't get that message. I don't think we need to use `auth_provider` at all. Usually set of usernames for regular auth and for plgrid is simply disjoint. So it's simpler to use only filter by user which cover 99% of cases.

If you really want to add `auth_provider` then I suggest following logic:
- `--user=a` filter only by username (will get the rows for user `a` from all user providers)
- `--auth-provider=x` filter only by auth provider (will get the rows for auth provider `x` )
- `--auth-provider=x --user=a` filter only by both

Reply:
I wanted make use of user_exists(name: str, auth_provider: str, users: db.Table, con) to give more feedback in case user doesn't exist. But of course I can do with both optional.

Reply:
If there is no such user, then print a message "such user doesn't exists" :)